src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
-public class SalaryInfo {
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class SalaryInfo extends CheckNotNull {
```

Comment:
I would say inheritance is not a case here, because it represents _'is-a'_ relationship. For sure, in this case `SalaryInfo` is not `CheckNotNull`